store/storetest/post_store.go

```diff
@@ -1408,25 +1409,25 @@ func testPostStoreGetFlaggedPostsForTeam(t *testing.T, ss store.Store, s SqlSupp
 
 	store.Must(ss.Preference().Save(&preferences))
 
-	r3 := (<-ss.Post().GetFlaggedPostsForTeam(o1.UserId, c1.TeamId, 0, 1)).Data.(*model.PostList)
+	r3, _ := ss.Post().GetFlaggedPostsForTeam(o1.UserId, c1.TeamId, 0, 1)
```

Comment:
I think the idea was to do it in all the places where it was happening, like here